app/routes/admin.places._index.tsx

```diff
@@ -41,10 +41,36 @@ export default function Route() {
             <li key={place.id}>
               <RemixLink
                 to={place.id}
-                className="card hover:card-hover block space-y-1"
+                className="card hover:card-hover block space-y-2"
               >
                 <h3>{place.name}</h3>
                 <code className="text-xs">{place.slug}</code>
+
+                <div className="queue-center text-xs">
+                  {place.isVerified ? (
+                    <Badge variant="brand" size="sm">
+                      <Check className="size-xs" />
+                      <span>Verified</span>
```

Comment:
```suggestion
                      <span>Verified by {place.verifiedBy.user.name}</span>
```

Shall we also show the user name here?

Reply:
Now the verifiedBy is not included when we select Place by Id.

So there is no detail data for that.

Reply:
Will exploring this later, since it needs to update the query as well.

Reply:
Thanks! For now, it's in an excellent state to be released as the first iteration of this feature.